runners/spark/job-server/build.gradle

```diff
@@ -82,13 +82,21 @@ runShadow {
     jvmArgs += ["-Dorg.slf4j.simpleLogger.defaultLogLevel=${project.property('logLevel')}"]
 }
 
-def portableValidatesRunnerTask(String name) {
+def portableValidatesRunnerTask(String name, Boolean streaming) {
+  def pipelineOptions = [
+          // Limit resource consumption via parallelism
+          "--parallelism=2",
```

Comment:
This option only applies to Flink, it has no meaning in Spark. https://github.com/apache/beam/blob/103facbf80c2e4786e60367161ec59352196d04a/runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java#L74